src/test/java/com/dokev/gold_dduck/event/service/EventServiceTest.java

```diff
@@ -60,4 +73,56 @@ void eventCreateTest() {
         assertNotNull(eventCode);
         assertThat(eventCode, is(newEvent.getCode()));
     }
+
+    @Test
+    @DisplayName("이벤트 코드 통한 이벤트 조회 테스트 - 성공")
+    void findDetailEventByCodeTest() {
+        //given
+        UUID eventCode = UUID.randomUUID();
+
+        Member member = TestMemberFactory.createTestMember();
+
+        Event event = TestEventFactory.builder(member)
+                .code(eventCode)
+                .build();
+
+        EventDto eventDto = makeEventDto(eventCode);
```

Comment:
TestEventFactory로 생성한 Event 엔티티와 makeEventDto로 생성한 EventDto는
이벤트 코드를 제외하고 내부값이 전부 달라서 사실 아예 다른 이벤트라고 할 수 있을 것 같습니다.
이 상태에서 테스트 코드가 통과하면 이 테스트가 올바르다고 할 수 없을 것 같습니다.

TestEventFactory로 만든 이벤트를 컨버터로 EventDto로 만들어 보시는건 어떤가요?

Reply:
```suggestion
        Event event = TestEventFactory.createEvent(testMember);
        
        EventDto eventDto = eventConverter.convertToEventDto(event);
```
약간 이런식으로 하면 makeEventDto이 없어도 될 것 같습니다.